ospf6d/ospf6_abr.c

```diff
@@ -155,37 +171,66 @@ int ospf6_abr_originate_summary_to_area(struct ospf6_route *route,
 	uint16_t type;
 	int is_debug = 0;
 
+	if (IS_OSPF6_DEBUG_ABR)
+		zlog_debug("%s : start area %s, route %pFX", __func__,
+			   area->name, &route->prefix);
+
+	if (route->type == OSPF6_DEST_TYPE_ROUTER)
+		summary_table = area->summary_router;
+	else
+		summary_table = area->summary_prefix;
+
+        summary = ospf6_route_lookup(&route->prefix, summary_table);
```

Comment:
Looks like bad formatting, would you mind running `clang-format` here?

You can run `clang-format` manually (less preferred) or call `git clang-format HEAD~1` (preferred, since it only touches where you changed).